catalog/spatial/geocoding/spatial-geocoding-query/src/test/java/org/codice/ddf/spatial/geocoding/query/TestGeoNamesQueryLuceneIndex.java

```diff
@@ -296,23 +294,22 @@ public void testNearestCitiesNullMetacard() {
 
     @Test(expected = IllegalArgumentException.class)
     public void testNearestCitiesNegativeRadius() {
-        directoryIndex.getNearestCities(new MetacardImpl(), -1, 1);
+        directoryIndex.getNearestCities("", -1, 1);
```

Comment:
The point of the test is to pass in a negative value for max_results and to verify that the IllegalArgumentException is thrown. So the first 2 parameters just need to pass validation, the actual values are irrelevant.